mmv1/products/compute/ResizeRequest.yaml

```diff
@@ -141,183 +130,49 @@ properties:
   - name: 'status'
     type: NestedObject
     description: |
-      [Output only] Status of the request.
+      Status of the request.
     output: true
     properties:
       # Status.error
       - name: 'error'
         type: NestedObject
         description: |
-          [Output only] Fatal errors encountered during the queueing or provisioning phases of the ResizeRequest that caused the transition to the FAILED state. Contrary to the lastAttempt errors, this field is final and errors are never removed from here, as the ResizeRequest is not going to retry.
+          Fatal errors encountered during the queueing or provisioning phases of the ResizeRequest that caused the transition to the FAILED state. Contrary to the lastAttempt errors, this field is final and errors are never removed from here, as the ResizeRequest is not going to retry.
         output: true
         properties:
           - name: 'errors'
-            type: NestedObject
```

Comment:
Why are these lines moved?

Reply:
As mentioned earlier, recently we have a issue (b/374273746) with lastAttemptErrors. [Test are failing](https://screenshot.googleplex.com/4L6xFjHW9qtrvLv) when trying to flatten resizeRequest lastAttemptErrors. It's because of enabling a new feature called ETA on lastAttemptError. It will add a details of estimate availability of the resource in near future. According to the defination [errors and errorDetails is a list of nested object](https://screenshot.googleplex.com/3cWVjwtzWe2WXwy), not a single object.  Even in mixer resource it defines the same way as you can see in the mentioned example. To fix the issue, I just modified them to an array of nested object. Since the lastAttemptError and StatusError use the same Error  (according to mixer definition) I modified them to a list of nested object too to avoid any future issues. That's the reason the lines moved one tab to the right directions.